session/pluginloader.go

```diff
@@ -112,13 +114,38 @@ func getUnixSocketDir(opts dgo.Map) string {
 	if v, ok := opts.Get("unixSocketDir").(dgo.String); ok {
 		return v.GoString()
 	}
-	if v := os.Getenv("TMPDIR"); v != "" {
+	if v := os.TempDir(); v != "" {
 		return v
 	}
 	return defaultUnixSocketDir
 }
 
-var defaultPluginTransport = "unix"
+// getDefaultPluginTransport returns the plugin transport method to use
+// for Windows this can be unix if the OS build is 17063 or above,
+// otherwise tcp is used
+// https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/
+func getDefaultPluginTransport() string {
+	// get the Windows build number from the registry
+	if runtime.GOOS == "windows" {
+		k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Windows NT\CurrentVersion`, registry.READ)
```

Comment:
I'm not able to build the project right now. A build fails with:
```
build golang.org/x/sys/windows/registry: cannot load golang.org/x/sys/windows/registry: no Go source files
```
I think the code dealing with the windows registry must be in a file that is build on windows only.